### PR TITLE
included fortrabbit in cloud.yml

### PR DESCRIPTION
**Pull request by frank-laemmer**
fortrabbit is a PHP hosting provider from and for Europe — shamless plug: i am co-founder.

### CODE REVIEW

**Comment by mxxcon** [comment]
missing a 32x32 logo
No need for double-empty lines between entries.
This server could not prove that it is www.fortrabbit.com; its security certificate is from *.eu1.frbit.net.

File: _data/cloud.yml [hunk 1]
```diff
@@ -46,6 +46,16 @@ websites:
           text: "Currently in beta as of June 5th 2014. Create a support ticket to request 2FA added to your organization"
       doc: https://support.cloud.engineyard.com/entries/40538256-Two-factor-authentication
 
+
+    - name: fortrabbit
+      url: https://www.fortrabbit.com/
+      twitter: fortrabbit.com
```
Comment (Carlgo11) [inline _data/cloud.yml:52]: twitter isn't needed for services already providing TFA to their customers.

File: _data/cloud.yml [hunk 2]
```diff
@@ -46,6 +46,16 @@ websites:
           text: "Currently in beta as of June 5th 2014. Create a support ticket to request 2FA added to your organization"
       doc: https://support.cloud.engineyard.com/entries/40538256-Two-factor-authentication
 
+
+    - name: fortrabbit
+      url: https://www.fortrabbit.com/
```
Comment (Carlgo11) [inline _data/cloud.yml:51]: invalid ssl/tsl cert. Either fix the certificate issue or change the url to http. (Though https is preferred)

**Comment by mxxcon** [comment]
Closed in favor of #1185.

**Pull request closed without merging**